src/components/SSidebarItem.vue

```diff
@@ -0,0 +1,61 @@
+<script>
+export default {
+  name: 'SSidebarItem'
+}
+</script>
+
+<script setup>
+const props = defineProps({
+  text: {
+    type: String,
+    required: true
+  },
+  icon: {
+    type: Object,
+    required: true
+  }
+})
+</script>
+
+<template>
+  <div class="sidebar-item">
+    <a href="/">
+      <h1 class="text"> {{ text }}
+       <component class="=link-icon" :is="icon" />
+      </h1> 
```

Comment:
Looks like you have a typo

```suggestion
      <h1 class="text"> {{ text }}
       <component class="link-icon" :is="icon" />
      </h1> 
```